pods-framework/pods
# Add copy button to field slugs to make it easier to copy/paste

## Description

Add a new `CopyButton` react component to be used along with field name. It allows user to copy the field name, and keyboard support styles are also considered.

## Related GitHub issue(s)

#7237 




## Testing instructions



1. Go to Edit Pods screen
2. Hover over field name or using keyboard to navigate to field wrapper, `Copy Button` (currently displayed as copy icon) should be visible and clickable.
3. Click copy button and the field name should be copied to your clipboard.

## Screenshots / screencast


https://github.com/pods-framework/pods/assets/75633537/40985757-0cd5-485a-95e5-4609fa4d5682

Using keyboard to copy field name.

https://github.com/pods-framework/pods/assets/75633537/5bc49bd1-b1b8-4044-a236-02f2904d9186




## Changelog text for these changes







Enhancement: Pods field name will now have a copy button near it to copy field name(slug). #7237  (@heybran) 

## PR checklist
